features/rt/manifests/05-kubeletconfig-worker-rt.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: machineconfiguration.openshift.io/v1
+kind: KubeletConfig
+metadata:
+  name: kubelet-worker-rt 
+spec:
+  machineConfigPoolSelector:
+    matchLabels:
+      worker-rt: ""
+  kubeletConfig: 
+    systemReserved:
+      cpu: 1000m
+      memory: 500Mi
+    kubeReserved:
+      cpu: 1000m
+      memory: 500Mi
+    cpuManagerPolicy: static
```

Comment:
Do you think we should configure the default NUMA behavior right here too? At least topologyManagerPolicy: best-effort so basic NUMA alignment with no rejections are enabled?

Reply:
maybe we can, but i think more additions would be good to add in followup PRs

Reply:
Just be sure that you add the topology manager feature gate as well or we can just do it in the following PR.